flash/data/batch.py

```diff
@@ -0,0 +1,138 @@
+from typing import Any, Callable, Mapping, Optional, Sequence, Union
+
+import torch
+from pytorch_lightning.trainer.states import RunningStage
+from pytorch_lightning.utilities.exceptions import MisconfigurationException
+
+from flash.data.utils import _contains_any_tensor, convert_to_modules
+
+
+class _Chainer(torch.nn.Module):
```

Comment:
`_DataChainer` or `_DataBatchChainer` wold be a more intuitive name, or add docs.

Reply:
just trying to figure out what does every class for now and missing some context

Reply:
You are right, we should find a better name.
`_TransformChainer` maybe